### PR TITLE
[WIP] Add import functionality to CodeBuild Project

### PR DESCRIPTION
**Pull request by ryof**
Though I couldn't find any issue about this, I made this PR because I'm using CodeBuild frequently.
I'm not familliar with Golang very well, so please point out if something is wrong.

**Changes proposed in this pull request:**

* Add import functionality to CodeBuild Project

**Output from acceptance testing:**

```
$ make testacc TESTARGS='-run=TestAWSCodeBuildProject_importBasic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v -run=TestAWSCodeBuildProject_importBasic -timeout 120m
?       github.com/terraform-providers/terraform-provider-aws   [no test files]
q=== RUN   TestAWSCodeBuildProject_importBasic
--- PASS: TestAWSCodeBuildProject_importBasic (48.20s)
PASS
ok      github.com/terraform-providers/terraform-provider-aws/aws       48.241s
```

### CODE REVIEW

File: aws/resource_aws_codebuild_project.go [hunk 1]
```diff
@@ -23,6 +23,9 @@ func resourceAwsCodeBuildProject() *schema.Resource {
 		Read:   resourceAwsCodeBuildProjectRead,
 		Update: resourceAwsCodeBuildProjectUpdate,
 		Delete: resourceAwsCodeBuildProjectDelete,
+		Importer: &schema.ResourceImporter{
```
Comment (bflad) [inline aws/resource_aws_codebuild_project.go]: Can you please document the new import support at the bottom of `website/docs/r/codebuild_project.html.markdown`? Feel free to copy-paste an existing `## Import` section from another resource and adjust appropriately.

File: aws/import_aws_codebuild_project_test.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+		CheckDestroy: testAccCheckAWSCodeBuildProjectDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSCodeBuildProjectConfig_basic(rName),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateVerifyIgnore: []string{
```
Comment (bflad) [inline aws/import_aws_codebuild_project_test.go]: I'm guessing this was copy-paste from another resource's testing? 😄  The attributes `force_destroy` and `acl` do not exist for this resource, so the whole `ImportStateVerifyIgnore` bit can be removed.

Comment (ryof) [reply aws/import_aws_codebuild_project_test.go]: Oh, I'm sorry. This is my first code-reading terraform, so I referenced s3 bucket or something...
There is no acl for code build indeed, I'm embarrassed. 😓

**Comment by bflad** [comment]
This has been released in [version 1.25.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1250-june-27-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Pull request merged**